### PR TITLE
Better code quality, same thing

### PR DESCRIPTION
**Pull request by JKereliuk**
:)

### CODE REVIEW

File: run/jenkins.py [hunk 1]
```diff
@@ -145,6 +130,9 @@ def main():
             '--log-mach=%s' % LOCAL_LOG_FILEPATH,
             '--log-wptreport=%s' % LOCAL_REPORT_FILEPATH,
         ]
+        if platform['browser_name'] == 'firefox':
+          # for webrtc
+          command.extend(['--setpref', 'media.navigator.streams.fake=true'])
```
Comment (rwaldron) [inline run/jenkins.py]: Nice and clean :)

**Comment by rwaldron** [comment]
I've just restarted the job that failed here: https://travis-ci.org/w3c/wptdashboard/jobs/302715424

**Comment by rwaldron** [comment]
I've just re-restarted both failed runs.

**Comment by mattl** [comment]
Approved. Restarting builds.

**Comment by rwaldron** [comment]
I've restarted both builds several times now and they continue to fail, but the failures don't appear to be related to this changeset.

**Pull request merged**